src/classes/RD2_ElevateIntegrationService_TEST.cls

```diff
@@ -634,8 +634,8 @@ public class RD2_ElevateIntegrationService_TEST {
             .withStatusClosed()
             .build();
         npe03__Recurring_Donation__c oldRd = newRd.clone();
-        newRd.LastElevateEventPlayed__c = 'event-v2';
-        oldRd.LastElevateEventPlayed__c = 'event-v1';
+        newRd.LastElevateVersionPlayed__c = 2;
```

Comment:
I think it's worth it to add new a method `withLastElevateVersionPlayed(Integer)` in TEST_RecurringDonationBuilder to update this field

Reply:
Good idea, I just pushed an update with this change.